app/models/grouping.rb

```diff
@@ -729,6 +730,7 @@ def revoke_repository_permissions
 
   # Removes repository permissions for a single StudentMembership object
   def revoke_repository_permissions_for_membership(student_membership)
+    $stderr.puts "In revoke"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.